### ISSUE TITLE
Snooze/Mute-All (locations) - Crashes Cyrano

### ISSUE THREAD

**Issue by cjcornell** [PropelVentures/Cyrano#65]
Snooze/Mute-All (locations) - Crashes Cyrano

**Comment by dakshatechnologies** [comment]
Snooze/mute all is done by kevin lee

**Comment by cjcornell** [comment]
no, this was not "all done" by Kevin Lee ... you worked on Mute/Snooze also!  particularly the internal mute/snooze operations.

**Comment by dakshatechnologies** [comment]
Mute / snooze work internally and its working well.but the . The the complete code of snooze all / mute all which is write by me but unfortunately its completely changed by kevin lee

**Comment by cjcornell** [comment]
ok thanks.  I will keep this open and remove your assignment for now.
